util/gen_effect_id.py

```diff
@@ -51,6 +51,14 @@
     "Flourishing Fan Dance": "1820",
     "Higanbana": "1228",
     "Wildfire": "861",
+    "Chain Stratagem": "1221",
+    "Vulnerability Up": "638",
+}
+
+# These costom name of effect will not be checked.
+# Use this only when you need to handle different effects with a same name.
+costom_mapping = {
+    "Embolden (Self)": "1239",
```

Comment:
I think it might be clearer if the CustomMapping just used the name directly rather than going through the `clean_name` function.  In other words, make this `EmboldenSelf: "1239",`

Reply:
Done. But I think a clean_name function here is necessary to prevent someone using this later add a not clean name.

Reply:
What about an assert/warning/error if a custom_mapping doesn't have a clean name?

Reply:
Fine, though I think this makes code longer?